demo/frontend/vite.config.ts

```diff
@@ -1,9 +1,19 @@
-import { defineConfig } from 'vite';
+import { resolve } from 'node:path';
 import Sonda from 'sonda/vite';
+import { defineConfig } from 'vite';
 
 // https://vite.dev/config/
 export default defineConfig({
   plugins: [Sonda({ enabled: false })],
+  resolve: {
```

Comment:
aliasing means we dont have to put the sdk in watch mode which is way easier and less resource-intensive.